pkg/trace/test/runner.go

```diff
@@ -109,18 +108,18 @@ func (s *Runner) Post(traceList pb.Traces) error {
 		return errors.New("post: trace-agent not running")
 	}
 
-	var buf bytes.Buffer
-	if err := msgp.Encode(&buf, traceList); err != nil {
+	bts, err := traceList.MarshalMsg(nil)
```

Comment:
Why this change? I don't think it matters here... Did it break?

Reply:
Yes, I removed the io.Writer based serializer so traceLIst no longer implements the `msg.Encodable` interface.